evalml/tests/component_tests/test_lgbm_regressor.py

```diff
@@ -289,3 +288,23 @@ def test_lgbm_regressor_nullable_type_incompatibility(
     sk_lgb = lgbm.sklearn.LGBMRegressor()
     sk_lgb.fit(X, y)
     sk_lgb.predict(X)
+
+
+@patch(
+    "evalml.pipelines.components.component_base.ComponentBase._handle_nullable_types",
+)
+def test_lgbm_calls_handle_nullable_types(
+    mock_handle_nullable_types,
+    X_y_binary,
```

Comment:
hmmm, I see your point. 

My one sticking point is just that we have tests that confirm the incompatibilitiy in the dependency still exists and is fixed by our handle method- test_<component>_nullable_type_incompatibility like you said - and tests that confirm our component can actually take in nullable types. So I view these tests as tying those two together - the reason we can pass those types in is because we call `_handle_nullable_types`. But I just checked, and removing any of the handle calls or the incompatibility properties cause tests to fail, so I think that you're right that these tests are overkill at this point.

Reply:
removed the tests 537b9f223